puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/katello-devel-forklift-test.yaml

```diff
@@ -4,6 +4,7 @@
     project-type: pipeline
     sandbox: true
     triggers:
+      - timed: "0 12 * * *"
```

Comment:
Jenkins can spread it out a bit so it's not exactly at minute 0:

```suggestion
      - timed: "H 12 * * *"
```

Reply:
Thanks! updated